README.md

```diff
@@ -85,11 +120,62 @@ usage: plot.py [-h] --savefile SAVEFILE
 optional arguments:
   -h, --help           show this help message and exit
   --savefile SAVEFILE  load trained model's state_dict from file
+  --pre_proc_method    {GMM, standard}
+                       specify the pre-processing method that you wish to employ
 ```
 
+#### Outputs Of The Model
+
+There are a selection of plots & metrics the model can output. These are given by parse arguments <b>--savemetrics</b>, <b>--savevisualisation</b> or functions <b>plot_elbo</b>, <b>plot_likelihood_breakdown</b>, <b>plot_variable_distributions</b> & <b>distribution_metrics</b> in the notebooks provided. These outputs give you a graph displaying the ELBO breakdown during training, the breakdown of categorical & numerical likelihoods, a comparison of variable distributions between original data & synthetic data for each variable as well as a csv file displaying all the distributional metrics from SDV.
```

Comment:
flags and names wrapped in \` e.g. <b>--savemetrics</b> -> `--save_metrics` (aligning with previous comment)